frontend/src/views/WheresMyGene/components/InfoPanel/components/RelativeGeneExpression/index.tsx

```diff
@@ -1,23 +1,50 @@
+import { Checkbox } from "czifui";
+import Image from "next/image";
 import { Content, Header, LowHigh } from "../../common/style";
-import { Dot, Dots, Wrapper } from "./style";
+import plasmaImage from "./plasma.png";
+import { ContentWrapper, StyledFormControlLabel, Wrapper } from "./style";
 
-const COLORS = ["#FFFFC1", "#FDD66D", "#F95929", "#CD0019", "#6C001D"];
+const CONTENT_WIDTH_PX = 100;
 
-export default function RelativeGeneExpression(): JSX.Element {
+interface Props {
+  handleIsScaledChange: () => void;
+  isScaled: boolean;
+}
+export default function RelativeGeneExpression({
+  handleIsScaledChange,
+  isScaled,
+}: Props): JSX.Element {
   return (
     <Wrapper>
       <Header>Relative Gene Expression</Header>
-      <Content>
-        <Dots>
-          {COLORS.map((color) => (
-            <Dot color={color} key={color} />
-          ))}
-        </Dots>
-        <LowHigh>
-          <span>0.0</span>
-          <span>1.0</span>
-        </LowHigh>
-      </Content>
+      <ContentWrapper>
+        <Content>
+          <Image
+            src={plasmaImage}
+            alt="visualization color scale: interpolatePlasma(1.0 - meanExpression)"
+            width={CONTENT_WIDTH_PX}
+          />
+          <LowHigh>
+            <span>0.0</span>
+            <span>{isScaled ? "1.0" : "6.0"}</span>
```

Comment:
Might not have the context here, why 6?

Reply:
My guess is that `6.0` is RankIt's max normalized value? @pablo-gar please help here 😆 Thank you!